io/www/src/fragment/corrector.js

```diff
@@ -1,4 +1,26 @@
 import { logDebug } from './common.js';
+const DATA_EXTRA_OPTIONS_REGEX = /data-extra-options="(\{[^}]*\})"/g;
+
+/**
+ * Fixes data-extra-options attributes in all relevant fields for adobe-home surface
+ * @param {object} context - Context object
+ */
+function fixAdobeHomeDataExtraOptions(context) {
+    if (context.surface !== 'adobe-home') {
```

Comment:
nit: maybe this could go before fixAdobeHomeDataExtraOptions?

Reply:
sure, will fix